components/status_dropdown/status_dropdown.jsx

```diff
@@ -105,6 +149,112 @@ export default class StatusDropdown extends React.PureComponent {
             </FormattedMessage>
         );
     }
+    handleClearStatus = (e) => {
+        e.stopPropagation();
+        this.props.actions.unsetCustomStatus();
+    };
+
+    onToggle = (open) => {
+        this.props.actions.setStatusDropdown(open);
+    }
```

Comment:
can be simplified with immediate return: 

```suggestion
    onToggle = (open) => this.props.actions.setStatusDropdown(open);
```